ksqldb-engine/src/test/java/io/confluent/ksql/materialization/ks/KsMaterializationFunctionalTest.java

```diff
@@ -167,6 +167,8 @@ public static void classSetUp() {
   public void setUp() {
     output = KsqlIdentifierTestUtil.uniqueIdentifierName();
 
+    final Random rand = new Random();
+    queryId = new QueryId("static" + rand.nextInt(1000));
```

Comment:
Might be a god idea to do something like what we do in Streams, to ensure a unique test while providing an id with some meaning for debugging purposes. Not necessarily right away, it's better to merge this ASAP to unblock the build -- but you can look at `IntegrationTestUtils#safeUniqueTestName` in Streams and see how it's used to get some idea of what we do

Reply:
This actually doesn't help with the failures in this test class, because we create the KafkaStreams application twice as part of execution (one is done in a "sandboxed" context first and this first execution still creates the Streams app, but doesn't run it) so even with unique names, all the tests would fail.

Reply:
Would it be possible to use a different query-id in the sandbox?